lemur/plugins/lemur_acme/challenge_types.py

```diff
@@ -243,6 +251,13 @@ def create_certificate(self, csr, issuer_options):
         pem_certificate, pem_certificate_chain = self.create_certificate_immediately(
             acme_client, dns_authorization, csr
         )
+
+        if "drop_last_cert_from_chain" in authority.options \
+                and authority.options.get("drop_last_cert_from_chain") is True \
+                and pem_certificate_chain.count("BEGIN CERTIFICATE") > 1:
```

Comment:
Does it make sense to move `pem_certificate_chain.count("BEGIN CERTIFICATE") > 1` into `drop_last_cert_from_chain` if it's common behavior?

Reply:
good point, revision will follow shortly